docs/about/overview.md

```diff
@@ -0,0 +1,50 @@
+
+(myst-ast)=
+
+**MyST Abstract Syntax Tree (AST)**. A bundle of parsed content that adheres to the MyST Specification. An [Abstract Syntax Tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree) is a way of structuring information according to a formal specification, so MyST Document Engines know how to parse content and _create_ an AST out of that content. Usually, a MyST AST takes the form of a `.json` data structure, with metadata attached to each piece of content that describes its role, relationships with other content, etc. Because MyST AST follows a formal structure, it can be **rendered** into many kinds of outputs (like HTML, PDF, docx, etc).
+
+Here's a diagram showing how these all relate to one another:
+
+:::{figure} ../images/myst-diagram.svg
+An overview of some major parts of the MyST stack and how they relate to one another.
+In the most common workflow, a text file written in MyST Markdown is parsed by the [MyST Document Engine](https://mystmd.org/guide) which outputs MyST Document that follows the [MyST Specification](https://mystmd.org/spec) (usually a JSON file). This MyST Document can be rendered into many different kinds of outputs, most-commonly HTML or PDF. This usually done by the MyST Document Engine, but can be done by any application that understands how to parse and use the MyST Document Specification.
+:::
+
+## Are there other MyST engines?
+
+The [MyST Specification](https://mystmd.org/spec) and [MyST Markdown Syntax](https://mystmd.org/guide) was designed so that others could implement their own parsers, document engines, and renderers using MyST. The best example of another MyST Document Engine is the MyST parser for Sphinx.
+
+The [MyST Parser for Sphinx](https://myst-parser.readthedocs.io) is an extension for [Sphinx](https://sphinx-doc.org) that can parse MyST Markdown into Sphinx's internal document structure. It was created to allow users to parse MyST Markdown syntax into Sphinx for [V1 of Jupyter Book](https://jupyterbook.org). It now exists as an independent extension for the Sphinx community, as Jupyter Book now uses the MyST Document Engine.
+
+The MyST Parser for Sphinx does not support all of the functionality that is defined in the MyST Specification - that's OK! Its goal is to be a useful tool for the Sphinx community that only leverages parts of the MyST ecosystem (the markdown flavor).
+
+## How does Jupyter Book relate to MyST?
+
+The MyST Markdown Syntax was originally created to allow [Jupyter Book V1](https://jupyterbook.org) to use the [Sphinx Document engine](https://sphinx-doc.org) with markdown content.
```

Comment:
This seems to say: "We created a new version of Markdown so we could use Markdown with Sphinx".

The actual motivation may be a bit tricky to explain, so also fine to leave as-is :)

Reply:
that is kind of true basically. Basically:

- Jupyter Book wanted to use Sphinx for its engine
- But Sphinx only spoke rST and we didn't want that
- And roles/directives didn't exist as a concept in a pre-existing markdown flavor
- So we created MyST as a Sphinx-native markdown flavor.